webview/src/experiments/components/App.test.tsx

```diff
@@ -784,7 +784,7 @@ describe('App', () => {
 
     const tooltip = screen.getByRole('tooltip')
 
-    expect(tooltip).toHaveTextContent('No filters applied')
+    expect(tooltip.textContent).toBe('No filters applied')
```

Comment:
This change effectively makes the test check that _only_ the given text is there, whereas `toHaveTextContent` will pass even if there is other text present.